### ISSUE TITLE
Find improvements

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#310]
The current implementation is very basic, it would be good to see these features in Ctrl+F:

- Display how many hits are found, like browsers do.
- Have "buttons" for next and previous.

So the line would look like:

```
search term                                        1 of 3    [∧] [∨]
```

**Comment by pslobo** [comment]
I agree with the number of occurrences. As for the buttons, I'm ok just using `enter`, `shift+enter` to move to next previous match.

**Comment by sebastianf** [comment]
on macOS you usually use cmg + g for finding the next search item

**Comment by pslobo** [comment]
Yep, that was almost my first instinct, then I remembered that cmd+g is the Go To function ;)
